<?xml version="1.0"?>
<launch>
	<node pkg="omnicopter_ros" type="motor_i2c_driver" name="motor_i2c_driver" output="screen"/>
	<node pkg="omnicopter_ros" type="motor_tester" name="motor_tester" output ="screen"/>
</launch>
